### ISSUE TITLE
The problem of loading pre-trained weights

### ISSUE THREAD

**Issue by NarcissuAsuka** [LiheYoung/ShrinkMatch#3]
Hi, I am a beginner, how do I load the pre-trained weights you provided in the relevant code

parser.add_argument('--checkpoint', default='', type=str, metavar='PATH',
                    help='path to latest checkpoint (default: none)')
parser.add_argument('-e', '--evaluate', dest='evaluate', action='store_true',
                    help='evaluate model on validation set')
parser.add_argument('--pretrained', default='', type=str, metavar='PATH', help='path to pretrained model')
                    help='path to pretrained model (default: none)')
parser.add_argument('---self-pretrained', default='', type=str, metavar='PATH', help='path to MoCoCo model (default: none)')
                    help='path to MoCo pretrained model (default: none)')
parser.add_argument('---super-pretrained', default='', type=str, metavar='PATH', help='path to supervised pretrained model (default: none)')
                    help='path to supervised pretrained model (default: none)')

Is the weight path set in --pretrained?

I'd appreciate it if you could take the time to answer.
